Modules/Microsoft365DSC/DSCResources/MSFT_EXOSharedMailbox/MSFT_EXOSharedMailbox.psm1

```diff
@@ -12,6 +12,10 @@ function Get-TargetResource
         [System.String]
         $PrimarySMTPAddress,
 
+        [Parameter()]
+        [System.String]
+        $Alias,
+
         [Parameter()]
         [System.String[]]
         $Aliases,
```

Comment:
Since this maps to the EmailAddresses property of the Set-Mailbox cmdlet, we should rename it to match the proper name. This would however be a breaking change, therefore we should probably add a new parameter EmailAddresses and keep the existing $Aliases one there, but mark it as deprecated and print a warning if users are attempting to use it instead of the EmailAddresses one.

Reply:
I agree that this would be a good change, but it's not within the scope of this PR.